docs/planning/bug_triage.md

```diff
@@ -0,0 +1,44 @@
+# Knative Eventing Bug Triage and Management
+
+This document summarizes how bugs will be triaged and managed in Knative Eventing WG.
+This is a supplement to [Sprint planning](sprintplanning.md) process.
+
+## Goal
+
+- Keep a low bug count. Bug count should not be consistently increasing release over release esp [priority/important-longterm](#Bug-priorities).
```

Comment:
In regards of `low bug count` and `increasing release over release` we need 2 things, this could be just TODOs in this doc
1. Reference of what low bug count means, 50 bugs is low, but 200 is high? 
2. We need to track this metric in what ever form from release to release, even a sheet in docs, could be a task in the template when starting a new release.

Reply:
I didn't try to put a number of "low". Ideally it should be "zero" but that is not practical. I would rather focus on trend release over release to make sure to make sure the rate declines. I wanted to try this approach for a couple of releases and then set the threshold in numbers.